packages/search/src/ago/get-items.ts

```diff
@@ -11,14 +12,47 @@ export async function getItems(
   authentication?: UserSession
 ): Promise<ISearchResult<IItem>> {
   const agoParams = encodeAgoQuery(params);
-  return searchItems({
-    ...agoParams,
-    params: {
-      token,
-      countFields: agoParams.countFields,
-      countSize: agoParams.countSize
-    },
-    portal,
-    authentication
-  });
+  if (agoParams.countFields) {
+    // if countFields are defined
+    // AGO allows only 3 countFields at max
+    const chunkedCountFields = chunkArray(
+      agoParams.countFields.split(","),
+      3
+    ).map(fieldArrayChunk => fieldArrayChunk.join(","));
+    let allCounts: Array<{
+      fieldName: string;
+      fieldValues: Array<{
+        value: any;
+        count: number;
+      }>;
+    }> = [];
+    let results;
+    for (const chunk of chunkedCountFields) {
```

Comment:
Minor: could just do a `Promise.all` with the searchItems requests and let the execution potentially happen in parallel